fields/date_picker/css/datepicker.css

```diff
@@ -101,7 +101,7 @@
 .cfdatepicker table tr td,
 .cfdatepicker table tr th {
   text-align: center;
-  width: 10px;
+  /*width: 10px;*/
```

Comment:
@New0 I'd rather we delete than comment out.

This is a nitpick -- IE it's fine, don't worry about it -- try and keep this in mind in the future.